src/tests/all.rs

```diff
@@ -280,6 +280,10 @@ fn new_req_with_keywords(app: Arc<App>, krate: Crate, version: &str,
     return req;
 }
 
+fn new_req_body_foo_2() -> Vec<u8> {
```

Comment:
Rename to say `version_2`; Otherwise it looks like one of those BS functions that differs slightly but who knows how.